content/en/docs/ops/integrations/spire/index.md

```diff
@@ -32,7 +32,7 @@ The integration is compatible with Istio upgrades.
 Istio provides a basic sample installation to quickly get SPIRE up and running:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml
```

Comment:
after this change, can we get the whole page test pass?
![image](https://user-images.githubusercontent.com/4354057/173485046-963021b9-b506-4013-9bbc-83b3522bf749.png)